ecs-cli/modules/utils/utils.go

```diff
@@ -106,3 +107,68 @@ func GetPartition(region string) string {
 		return "aws"
 	}
 }
+
+// ParseLoadBalancers parses a StringSlice array into an array of load balancers struct
+// Input: ["targetGroupArn="...",containerName="...",containerPort=80","targetGroupArn="...",containerName="...",containerPort=40"]
+func ParseLoadBalancers(flagValues []string) ([]*ecs.LoadBalancer, error) {
+	var list []*ecs.LoadBalancer
+
+	for _, flagValue := range flagValues {
+		m := make(map[string]string)
+		validFlags := map[string]bool{
+			"containerName": false,
+			"containerPort": false,
+		}
+		var elbv1, elbv2 bool
+		keyValPairs := strings.Split(flagValue, ",")
+
+		for _, kv := range keyValPairs {
+			pair := strings.SplitN(kv, "=", -1)
+			if len(pair) > 2 {
+				return nil, fmt.Errorf("Only include one = to indicate your value in your %s", pair[0])
+			}
+			if pair[0] != "targetGroupArn" && pair[0] != "loadBalancerName" && pair[0] != "containerName" && pair[0] != "containerPort" {
```

Comment:
Would be better to define key, val := pair[0], pair[1] to make it clearer. Also can you make "targetGroupArn"... those to be constant vars?